moveos/moveos-stdlib/moveos-stdlib/sources/move_module.move

```diff
@@ -101,14 +140,36 @@ module moveos_std::move_module {
 
     native fun check_compatibililty_inner(new_bytecodes: vector<u8>, old_bytecodes: vector<u8>);
 
-    /// Native function to remap addresses in module binary where the length of
-    /// `old_addresses` must equal to that of `new_addresses`.
-    native fun remap_module_addresses_inner(
+    /// Native function to replace addresses identifier in module binary where the length of
+    /// `old_addresses` must equal to that of `new_addresses`.  
+    native public(friend) fun replace_address_identifiers(
+        bytes: vector<vector<u8>>,
+        old_addresses: vector<address>,
+        new_addresses: vector<address>,
+    ): vector<vector<u8>>;
+
+    /// Native function to replace the name identifier `old_name` to `new_name` in module binary.
+    native public(friend) fun replace_identifiers(
+        bytes: vector<vector<u8>>,
+        old_idents: vector<String>,
+        new_idents: vector<String>,
+    ): vector<vector<u8>>;
+
+    /// Native function to replace constant addresses in module binary where the length of
+    /// `old_addresses` must equal to that of `new_addresses`.    
+    native public(friend) fun replace_addresses_constant(
         bytes: vector<vector<u8>>,
         old_addresses: vector<address>,
         new_addresses: vector<address>,
     ): vector<vector<u8>>;
 
+    /// Native function to replace constant bytes in module binary where the length of
+    /// `old_addresses` must equal to that of `new_addresses`.    
```

Comment:
typo in comment : `old_addresses` -> `old_bytes`, `new_addresses` -> `new_bytes`